srcpkgs/python3-dnsrecon/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'python3-dnsrecon'
 pkgname=python3-dnsrecon
-version=1.1.4
+version=1.1.5
 revision=2
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3 python3-netaddr python3-dnspython python3-lxml python3-flake8 python3-setuptools"
+depends="python3 python3-netaddr python3-dnspython python3-lxml python3-flake8 python3-setuptools python3-pyflakes"
```

Comment:
None of `python3-flake8`, `python3-pyflakes` or `python3-setuptools` belong in `depends`.

Reply:
Fixed that, will push shortly, still trying to figure out what to do about some zone transfer failing, perhaps just patch to remove some of that?